### PR TITLE
release manager role

### PR DESCRIPTION
**Pull request by justinengland**
adding release manager role

### CODE REVIEW

File: collections/_job_descriptions/06-release-manager.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+order: 6
+title: "Release Manager"
+---
+
+Chia network is looking to hire a release manager, primarily focused on curation of the Git release process for all chia blockchain software releases.
```
Comment (pmaslana) [inline collections/_job_descriptions/06-release-manager.md:6]:
```suggestion
Chia Network is looking to hire a release manager, primarily focused on the curation of Git release processes for all Chia-Blockchain software releases.
```

File: collections/_job_descriptions/06-release-manager.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+The ideal candidate for this role has experience leading software releases in a multi team, multi language environment. This candidate will also have expert experience using Git and an expert level understanding of Git processes.
```
Comment (pmaslana) [inline collections/_job_descriptions/06-release-manager.md:8]:
```suggestion
The ideal candidate for this role has experience leading software releases in a multi-team, multi-language environment. This candidate will also have significant experience using Git and an expert-level understanding of Git processes.
```

File: collections/_job_descriptions/06-release-manager.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+Successful candidates will be able to articulate a vision for Git process management and may even have a preferred practice they have used successfully in previous roles. Including an ability to articulate what the benefits and flaws are in your preferred practice.
```
Comment (pmaslana) [inline collections/_job_descriptions/06-release-manager.md:10]:
```suggestion
Successful candidates will be able to articulate a vision for Git process management. Candidates may provide a preferred practice they have successfully used in previous roles, including an ability to elaborate on the benefits and flaws are in your preferred practice.
```

File: collections/_job_descriptions/06-release-manager.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+* Manage release approval process.
+* Establish a strong understanding of each release and the code changes contained within.
+* Ownership of scheduling and cadence of releases.
+* Enforce Git process best practices.
+* Ownership of release process.
+
+**Release Manager Requirements**
+
+* Expert level knowledge of Git.
```
Comment (pmaslana) [inline collections/_job_descriptions/06-release-manager.md:22]:
```suggestion
* Expert-level of knowledge of Git.
```

File: collections/_job_descriptions/06-release-manager.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+* Experience releasing open source software projects.
+* Experience gathering stakeholder consensus and running CAB style release meetings.
+* Solid engineering background with a base level understanding of software development.
```
Comment (pmaslana) [inline collections/_job_descriptions/06-release-manager.md:25]:
```suggestion
* Solid engineering background with a base-level understanding of software development.
```

File: collections/_job_descriptions/06-release-manager.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+**Perks**
+
+Chia Network Inc. allows for flexible work hours and remote work for this position. Core hours should match United States working hours. We offer competitive salaries and a comprehensive benefits package including medical, dental, and life insurance for you, your spouse/domestic partner and children.
```
Comment (pmaslana) [inline collections/_job_descriptions/06-release-manager.md:29]:
```suggestion
Chia Network Inc. allows for flexible work hours and remote work for this position. Core hours should match United States working hours. We offer competitive salaries and a comprehensive benefits package including medical, dental, and life insurance for you, your spouse/domestic partner, and children.
```

File: collections/_job_descriptions/06-release-manager.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+**Application Process**
+
+Send your CV/resume along with your Keybase name (if available) to jobs@chia.net, please include the position title as your subject line
```
Comment (pmaslana) [inline collections/_job_descriptions/06-release-manager.md:33]:
```suggestion
Send your CV/resume along with your Keybase name (if available) to jobs@chia.net, please include the position title as your subject line.
```

**Review by hoffmang9 — APPROVED** [review]
Lgtm

**Comment by hoffmang9** [comment]
Apologies that my review crossed with your review @pmaslana

**Comment by pmaslana** [comment]
@hoffmang9 No problem! I must have taken a little too long to review.

**Pull request merged**